### PR TITLE
Remove requirement of cloud symbol

### PR DESCRIPTION
**Pull request by ghost**
When referencing a cloud variable, you no longer need to use the '☁ ' symbol. However, you can still use '☁ ' if you want to.

### CODE REVIEW

File: cloud.js [hunk 1]
```diff
@@ -70,15 +70,15 @@ class Cloud {
   }
   set(name, val) {
     this.eventEmitter.emit("setByServer", name, val);
-    this._send('set', {name:name,value:val});
+  this._send('set', {name:name.startsWith('☁ ')?name:'☁ '+name,value:val});
```
Comment (AmazingMech2418) [inline cloud.js]: This should be indented...

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "scratchcloud",
-  "version": "1.0.7",
+  "version": "1.0.8",
```
Comment (AmazingMech2418) [inline package.json]: Due to a feature change, a shift to 1.1.0 would probably be better. (third digit is a patch, second is a feature, and first is a refactor)

**Review by AmazingMech2418 — CHANGES_REQUESTED** [review]
Seems good! Just a few minor updates that would be better.

**Comment by ghost** [comment]
I have resolved the issues.

**Review by AmazingMech2418 — APPROVED** [review]
Looks good!

**Pull request merged**